bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/async/BulkMutation.java

```diff
@@ -442,10 +447,48 @@ public void run() {
         }, autoflushMs, TimeUnit.MILLISECONDS);
       }
     }
-
     return future;
   }
 
+  private ListenableFuture<MutateRowResponse> processUnsafeMutationSplit(MutateRowsRequest.Entry entry) {
+    final SettableFuture<MutateRowResponse> mutateRowResponseListenableFuture = SettableFuture.create();
+    final List<List<Mutation>> subSets = Lists.partition(entry.getMutationsList(), (int) MAX_NUMBER_OF_MUTATIONS);
+    // split subSets into several entries of 100,000
+    for (List<Mutation> mutations : subSets) {
+      ListenableFuture<MutateRowResponse> childFuture = sendDirect(entry.getRowKey(), mutations);
+      Futures.addCallback(childFuture, new FutureCallback<MutateRowResponse>() {
```

Comment:
This callback needs to be created outside the for loop, otherwise, `success` will never be more than 1.